who_are_we.md

```diff
@@ -0,0 +1,63 @@
+# Who Are We
+###The Amitai and Brice Story:
+
+A couple of guys, with a pretty interesting perspective, joined forces, synergized, made a lot of mistakes, learned and repeated.  They like the idea of open source, in fact they like the idea of open everything. After plenty of failures and pretty amazing success, these guys, with the help of this team, are still paving the way in the interesting world of web development.
+
+###The Gizra Way:
+
+The Gizra story is about a way, an outlook, a philosophy that navigates all the internal and external aspects of a web development company. It is knowing and acting on the understanding that *we can do better.*
+
+It is summed up nicely under four sections of The Gizra Way. Let’s look at them:
+
+**Transparency**.  Everything is out in the open, warts and all.  This may be new to you and may take some adjustment. We set the tone as early as our price estimations by basing them on a detailed breakdown of project tasks (small chunks of 12 hours or less) with an attached cost.  This extends into our projects where our clients have full access to Github. They see everything - including our internal communications. Even this book is public. 
```

Comment:
I think that apart of `Transparency`, the other points have a too markety tone to them that we'll need to tone down. The audience are new employees that need learn about our nerative. For example `Meticulous Execution` is a bit of BS ;) - something like "We try to keep high standards" is probably more realistic.